src/main/java/org/example/ReadingFileAllNumberOfCharacters.java

```diff
@@ -0,0 +1,24 @@
+package org.example;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static java.lang.Character.isDigit;
+import static java.lang.Character.isJavaLetter;
+
+public class ReadingFileAllNumberOfCharacters {
+    public static void printAllNumberOfCharacters(File file) throws IOException {
+        String pathToFile = file.getPath();
+        String text = Files.readString(Path.of(pathToFile));
+        char[] charArray = text.toCharArray();
+        int count = 0;
+        for (char c : charArray) {
+            if (isJavaLetter(c) || isDigit(c)) {
```

Comment:
a jeśli w tekście będzie '.' albo '?' albo spacja to dlaczego tego nie liczysz? nie wiem czy nie lepszym i prostszym podejściem nie było by zrobienie text.length